packages/react/ds/src/checkbox/types.ts

```diff
@@ -0,0 +1,23 @@
+import { InputHTMLAttributes } from 'react';
+
+export const CheckboxSizeEnum = {
+  Large: 'lg',
+  Medium: 'md',
+  Small: 'sm',
+} as const;
+
+export type CheckboxSizeEnumType =
```

Comment:
I think it's better to not add additional types, you can use the same name
```suggestion
export type CheckboxSizeEnum =
```

Reply:
using a const object doesn’t automatically give us a type like an enum does.... I mean you can use enum as type and const at the same time.

 so we do need a separate type definition, we can reuse the same name  “name shadowing” (having a const and type with the same name), but having a separate name keeps it clearer that one is a runtime object and the other is a type..